src/sidebar/util/service-context.js

```diff
@@ -64,7 +64,7 @@ export function withServices(Component) {
   function Wrapper(props) {
     // Get the current dependency injector instance that is provided by a
     // `ServiceContext.Provider` somewhere higher up the component tree.
-    const $injector = useContext(ServiceContext);
+    const injector = useContext(ServiceContext);
```

Comment:
I removed the `$` here because this no longer refers to Angular's `$injector` service but an instance of the `Injector` class from `src/shared/injector.js` instead.